app/Console/TaskOverdueNotificationCommand.php

```diff
@@ -50,6 +60,54 @@ public function showTable(OutputInterface $output, array $tasks)
     }
 
     /**
+     * Send all overdue tasks for one user in one email
+     *
+     * @access public
+     */
+    public function sendGroupOverdueTaskNotifications()
+    {
+        $tasks = $this->taskFinder->getOverdueTasks();
+
+        foreach ($this->groupByColumn($tasks, 'owner_id') as $user_tasks) {
+            $users = $this->userNotification->getUsersWithNotificationEnabled($user_tasks[0]['project_id']);
+
+            foreach ($users as $user) {
+                $this->sendUserOverdueTaskNotifications($user, $user_tasks);
+            }
+        }
+
+        return $tasks;
+    }
+
+    /**
+     * Send all overdue tasks in one email to project manager(s)
+     *
+     * @access public
+     */
+    public function sendOverdueTaskNotificationsToManagers()
+    {
+        $tasks = $this->taskFinder->getOverdueTasks();
+
+        foreach ($this->groupByColumn($tasks, 'project_id') as $project_id => $project_tasks) {
+            $users = $this->userNotification->getUsersWithNotificationEnabled($project_id);
+
+            $managers = array();
+            foreach ($users as $user) {
+                $role = ProjectUserRole::getUserRole($project_id, $user['id']);
```

Comment:
You cannot call this method statically, instead use `$this->projectUserRole->getUserRole(...`